pom.xml

```diff
@@ -144,110 +240,6 @@
                     <url>https://repo.mrivanplays.com/repository/od-snapshots/</url>
                 </snapshotRepository>
             </distributionManagement>
-            <build>
-                <defaultGoal>clean package</defaultGoal>
-                <finalName>
-                    ${project.artifactId}-${plugin.version}-${git.commit.id.abbrev}-${plugin.specificationVersion}
```

Comment:
Is this `finalName` specified anywhere else?

Reply:
That has nothing to do with this scope of this pr

Reply:
I noticed it was removed, is the functionality still available?

Reply:
Are you blind or sth??

Reply:
Couldn't see an equivalent when I was browsing the files of this PR.

Reply:
You are blind confirmed.